src/pybind/mgr/dashboard_v2/README.rst

```diff
@@ -83,7 +83,71 @@ is located)::
 
   $ tox
 
+
 If you just want to run a single tox environment, for instance only run the
 linting tools::
 
   $ tox -e lint
+
+Developer Notes
+_______________
```

Comment:
For the sake of consistency, could you please use dashes not underlines for marking Level 2 headings?

Reply:
sure